contrib/babelfishpg_common/src/collation.h

```diff
@@ -5,6 +5,14 @@
 #include "nodes/nodeFuncs.h"
 #include "nodes/pathnodes.h"
 
+
+extern char *pltsql_default_locale;
+extern char *pltsql_server_collation_name;
```

Comment:
I may miss something because of large diff. Would you let me know where these extern variables are used?

Reply:
These are global variable being used to define server_collation_name and default_locale GUCs.